### DISCUSSION TITLE
Deconstructor used by bots on stations wont transfer dropped items

### DISCUSSION THREAD

**Discussion by BiggestZocker** [FakeFishGames/Barotrauma#15454] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I told bots to deconstruct ores on a station. A hint came saying items left behind by bots or in the deconstructor would be transfered upon departure. But the bots deconstructed more items than the 5 deconstructor slots could hold, so items dropped on the ground. On depature, the items dropped from the deconstructor were not tranfered to the ship. This is very unintuitive

### Reproduction steps

1. Dock a station
2. Use commands to tell bots to deconstruct several items (more than 5 stacks of products)
3. Leave station

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
